.github/workflows/integration-build-product.yml

```diff
@@ -877,6 +939,7 @@ jobs:
           echo "msg_error=${MESSAGE_ERR}\n  Details: ${{ steps.job_info.outputs.job_url }}" >> $GITHUB_OUTPUT
 
       - name: Slack message success
+        if: ${{ github.event_name != 'pull_request' }}
```

Comment:
please post a comment on the PR which indicates at least:
-  artifacts were generated and available at url: https://github.com/MOV-AI/project-tugbot2/actions/runs/10740271564 for example
- artifacts will expire in x days or after x operation
- how robot.json can be used to test locally

see: https://github.com/peter-evans/create-or-update-comment

Reply:
done